File: swift/sprint5/K/solution.swift
/*
 Comment it before submitting

 open class Node {
     
     public var value: Int
     public var left: Node?
     public var right: Node?
     
     public init(_ value: Int) {
         self.value = value
         self.right = nil
         self.left = nil
     }
 }
 */

func printRange(root: Node?, from: Int, to: Int) {
    // Your code
    // ヽ(´▽`)/
}

func test() {
    var node1 = Node(2)
    var node2 = Node(1)
    node2.right = node1
    var node3 = Node(8)
    var node4 = Node(8)
    node4.right = node3
    var node5 = Node(9)
    node5.left = node4
    var node6 = Node(10)
    node6.left = node5
    var node7 = Node(5)
    node7.left = node2
    node7.right = node6
    printRange(root: node7, from: 2, to: 8);
    // expected output: 2 5 8 8
}

File: csharp/sprint5/K/Solution.cs
using System.IO;

#if REMOTE_JUDGE
#else
public class Node
{
    public int Value { get; set; }
    public Node Left { get; set; }
    public Node Right { get; set; }

    public Node(int value)
    {
        Value = value;
        Left = null;
        Right = null;
    }
}
#endif

public class Solution
{
    public static void PrintRange(Node root, int left, int right, StreamWriter writer)
    {
        // Your code
        // “ヽ(´▽｀)ノ”
    }
}
